motor_gain: 1.35
wheel_radius: 0.0331
wheel_sep: 0.1058
